notebooks/source/ordinalregression.ipynb

```diff
@@ -0,0 +1,239 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Ordinal regression with the ImproperUniform distribution"
```

Comment:
I think it is better to use "Ordinal Regression" for further enhancements in the future. For example, it is not necessary to restrict ourselves to "improper" world, where we can add any extra log prob terms to the joint log probability (as in the `model2`) to create an "unnormalized" distribution. I can add a "proper" version in a follow-up PR to clarify that.